modules/02_welfare/ineqLognormal/equations.gms

```diff
@@ -307,8 +307,7 @@ q02_inconvPenCoalSolids(t,regi)$(t.val > 2005)..
     v02_inconvPenCoalSolids(t,regi)
   =g=
 *' local air pollution for coal: inconvinienve penalty applies only for buildings use; slack variable ensures that v02_inconvPen can stay > 0
-    p02_inconvpen_lap(t,regi,"coaltr") * (vm_prodSe(t,regi,"pecoal","sesofos","coaltr")
-  - vm_cesIO(t,regi,"fesoi"))
```

Comment:
Should this rather be a sum `fesoi ≡ feso_cement + feso_chemicals + feso_steel + feso_otherIn`?

Reply:
@JakobBD Are you aware of this? Looks like air pollution inconvenience penalty is reduced by industry and only applied for buildings. Can we delete this, or do we have to rewrite the term?